src/scss/components/navigation/_index.scss

```diff
@@ -93,3 +99,28 @@
     }
   }//hx-reveal
 }
+
+// ===== SUB-LEFT NAVIGATION =====
+#sub-leftNav, #section-1, #section-1-2 {
+  > a:active, > a:focus {
+    border-left: 0.125rem solid $purple-500;
+    color: $purple-500;
+    font-weight: 600;
+    outline: none;
+  }
+}
+
+#sub-leftNav a:active, #sub-leftNav a:focus {
+  margin-left: 0.3rem;
+  padding-left: 0.8rem;
+}
+
+#section-1 > a:active, #section-1 > a:focus {
+  margin-left: 1.25rem;
+  padding-left: 1.063rem;
+}
+
+#section-1-2 > a:active, #section-1-2 > a:focus {
+  margin-left: 2.5rem;
+  padding-left: 1.125rem;
+}
```

Comment:
@badejayayesubabu let's chat about this approach.  Normally, we avoid using `id`s.

Reply:
Changes have been done. Please review